File: SEP3_Client/Model/Unit/Time/MyTime.cs
using System;

namespace SEP3_Client.Model.Unit.Time
{
    public class MyTime
    {
        public int Year { get; set; }
        public int Month { get; set; }
        public int Day { get; set; }
        public int Hour { get; set; }
        public int Minute { get; set; }

        public MyTime(DateTime dateTime)
        {
            Year = dateTime.Year;
            Month = dateTime.Month;
            Day = dateTime.Day;
            Hour = dateTime.Hour;
            Minute = dateTime.Minute;
        }

        private MyTime(MyTime time)
        {
            Year = time.Year;
            Month = time.Month;
            Day = time.Day;
            Hour = time.Hour;
            Minute = time.Minute;
        }
        
        public MyTime(){}

        public MyTime Copy()
        {
            return new MyTime(this);
        }

        public override string ToString()
        {
            return Day + "/" + Month + "/" + Year + "-" + Hour + ":" + Minute;
        }
    }
}

File: SEP3_Database/src/main/java/com/example/SEP3_Database/model/domain/unit/time/MyTime.java
package com.example.SEP3_Database.model.domain.unit.time;

import javafx.scene.shape.HLineTo;

import java.sql.Timestamp;
import java.time.LocalDateTime;

public class MyTime {
    private int Year;
    private int Month;
    private int Day;
    private int Hour;
    private int Minute;

    public MyTime(LocalDateTime dateTime)
    {
        Year = dateTime.getYear();
        Month = dateTime.getMonthValue();
        Day = dateTime.getDayOfMonth();
        Hour = dateTime.getHour();
        Minute = dateTime.getMinute();
    }

    public MyTime(int year,int month,int day,int hour,int minute)
    {
        Year = year;
        Month = month;
        Day = day;
        Hour = hour;
        Minute = minute;
    }

    public int getMinute() {
        return Minute;
    }

    public void setMinute(int minute) {
        Minute = minute;
    }

    public int getHour() {
        return Hour;
    }

    public void setHour(int hour) {
        Hour = hour;
    }

    public int getDay() {
        return Day;
    }

    public void setDay(int day) {
        Day = day;
    }

    public int getMonth() {
        return Month;
    }

    public void setMonth(int month) {
        Month = month;
    }

    public int getYear() {
        return Year;
    }

    public void setYear(int year) {
        Year = year;
    }

    public String toString()
    {
        return Day + "/" + Month + "/" + Year + "-" + Hour + ":" + Minute;
    }

    public MyTime copy()
    {
        return new MyTime(Year,Month,Day,Hour,Minute);
    }

    public void setThroughDateTime(Timestamp dateTime)
    {
        int Y = dateTime.getYear();
        int M = dateTime.getMonth();
        int D = dateTime.getDay();
        int H = dateTime.getHours();
        int Mi = dateTime.getMinutes();
        Year = Y;
        Month = M;
        Day = D;
        Hour = H;
        Minute = Mi;
    }
}
